spec/support/factory_bot.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+end
```

Comment:
i guess this should be in spec_helper, is not it?

Reply:
I can't remember why I did it, but based on the [article](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#rspec), this file should be, but you need to redo rails_helper a little.